pkg/metallb/metallb.go

```diff
@@ -336,6 +336,32 @@ func (builder *Builder) WithOptions(options ...AdditionalOptions) *Builder {
 	return builder
 }
 
+// WithFRRconfigAlwaysBlock adds specific routes to block from being advertised to the FRR nodes.
+func (builder *Builder) WithFRRconfigAlwaysBlock(prefixes []string) *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Adding prefixes to block in the metallb.io object %s", builder.Definition.Name)
+
+	if len(prefixes) < 1 {
+		builder.errorMsg = "can not accept empty prefix list for the metallb alwaysBlock mode"
+	}
+
+	if builder.errorMsg != "" {
+		return builder
+	}
+
+	if builder.Definition.Spec.FRRK8SConfig == nil {
+		builder.Definition.Spec.FRRK8SConfig = &mlbtypes.FRRK8SConfig{}
+	}
+
+	builder.Definition.Spec.BGPBackend = "frr-k8s"
```

Comment:
I think, we should validate that metallb in "frr-k8s" mode, but not configure it

Reply:
Lets discuss it this evening

Reply:
?

Reply:
Correct this is default. I removed it